utils/windows_configuration.py

```diff
@@ -17,13 +17,29 @@
 
 log = logging.getLogger(__name__)
 
+config_attributes = ['api_key', 'tags', 'hostname', 'proxy_host', 'proxy_port', 'proxy_user', 'proxy_password']
+
+def remove_registry_conf():
+    try:
+        with _winreg.OpenKey(_winreg.HKEY_LOCAL_MACHINE,
+                             WINDOWS_REG_PATH, 0, _winreg.KEY_WRITE) as reg_key:
+            for attribute in config_attributes:
+                try:
+                    _winreg.DeleteValue(reg_key, attribute)
+                except Exception as e:
```

Comment:
Will it error if the key doesn't exists? In that case, it seems like we would want to ignore that error instead of logging it.